### ISSUE TITLE
Give the option to add diagram dugga to a course during the install #6849

### ISSUE THREAD

**Issue by a17adajo** [HGustavs/LenaSYS#6612]
It would be nice if the diagram was available somewhere after a fresh install

**Comment by HGustavs** [comment]
Great suggestion.

This also applies to many other views etc

There should be a test course that is used only for showcasing the various features.

This way we know we can go to the showcase course to demo any features.

During the presentation, more than once the presentation was held up by people looking for "their" view in some course.

I say we create a global "demo" or "showcase" course with all features present.

**Comment by a18kaswi** [comment]
Check this when #6790 is solved

**Comment by a18kaswi** [comment]
R.n it boils down to how to beautifuly structure an interactive function that mods the install-db during install.

**Comment by a18kaswi** [comment]
Dug more, need to

1. Construct 'Diagram-Dugga' - \htdocs\LenaSYS\DuggaSys\templates

2. Implement the ability to install 'Demo-Course' 

3. Add 'Diagram-Dugga' to 'Demo-Course'.

**Comment by a18kaswi** [comment]
diagram_dugga.js
diagram_dugga.html

Available from the demo-page implemented in #8744 

these files are just placeholder tho.

So now we are down to 

1. Construct 'Diagram-Dugga' - \htdocs\LenaSYS\DuggaSys\templates

**Comment by a18kaswi** [comment]
The DiagramDuggais now available in the Demo-Course.

Note that this is a placeholder and that the actual dugga needs to be created. This will be a new issue.
